pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "aapns"
+version = "19.9"
+description = "Asynchronous Apple Push Notification Service Client"
+authors = ["Jonas Obrist <jonas.obrist@hennge.com>"]
+license = "Apache-2.0"
+repository = "https://github.com/HDE/aapns"
+
+[tool.poetry.dependencies]
+python = "^3.7"
```

Comment:
I think that `pypi` could still benefit from classifiers, because this python requirement is not automatically converted to those.

Example: https://github.com/dimaqq/awaitwhat/blob/80ecc1e74a934e02530f13e44f7dbda1a225f5a4/pyproject.toml#L17-L20